packages/lesswrong/lib/collections/comments/fragments.ts

```diff
@@ -74,6 +74,17 @@ registerFragment(`
     rejectedReason
     modGPTRecommendation
     originalDialogueId
+    doppelComments {
```

Comment:
This is going to have a pretty substantial performance impact, in a place that isn't forum-gated